lte/cloud/go/services/subscriberdb/obsidian/models/conversion.go

```diff
@@ -144,8 +144,9 @@ func (m *MutableSubscriber) FromEnt(ent configurator.NetworkEntity, policyProfil
 		model.StaticIps = config.StaticIps
 		// If no profile in backend, return "default"
 		// TODO(8/21/20): enforce this at the API layer (and include data migration)
-		if model.Lte.SubProfile == "" {
-			model.Lte.SubProfile = "default"
+		if model.Lte.SubProfile != nil && *model.Lte.SubProfile == "" {
```

Comment:
if model.Lte.SubProfile == nil || *model.Lte.SubProfile == "" {  ...

Reply:
hey @emakeev , you can use the <code>```suggestion</code> block, to make it very easy to see the changes you are suggesting:

```suggestion
		if model.Lte.SubProfile != nil || *model.Lte.SubProfile == "" {
```